src/components/Facility/ConsultationDetails/QuestionnaireResponsesList.tsx

```
@@ -453,50 +534,97 @@ function ResponseCardContent({ item }: { item: QuestionnaireResponse }) {
+            )}
+            <div className="ml-auto flex items-center gap-1">
+              {isEnteredInError && (
+                <ChevronDown
+                  className={cn(
+                    "size-4 transition-transform text-gray-500",
+                    isExpanded && "rotate-180",
+                  )}
+                />
+              )}
+              {!isPrintPreview && (
+                <div onClick={(e) => e.stopPropagation()}>
```

Biome (2.3.14)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)